prognoses_monitoring_reports_code/automatic_checks/automatic_sanity_check.py

```diff
@@ -85,3 +90,56 @@ def check_if_predicted_is_suspicious(
         report.add_check_failed(reason="Predicted is series of zeros")
 
     return report
+
+
+def check_quality_sufficient(
+    realised: pd.Series, predicted: pd.Series, ean: int
+) -> CheckReport:
+    """Simple function to check the content of t-forecasts for e certain minimal quality
+
+    Args:
+        realised: realised values
+        predicted: predicted value
+        ean: ean of specific connection
+
+    Returns: Checkreport object that indicates if any checks failed and for what reason.
+
+    """
+
+    # Initialize report
+    report = CheckReport(ean)
+
+    # Format data to fit the metric functions in openSTEF
+    realised = realised.rename("load")
+
+    # Create a basecase forecast by shifting the realised load 7 days (7*96 PTU's)
+    basecase = realised.copy(deep=True).rename("basecase")
+    basecase.index = basecase.index - (7 * 96)
```

Comment:
This does not work for datetime indexed dataframes, use something like `timedelta(days=7)` instead.